desktop/cmp/input/CalendarDateInput.js

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
I wonder if we could avoid duplicating all of our DateInput work, and just add an option to the existing DateInput to get it to bind to a CalendarDate instead of a Date?